docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -231,8 +231,15 @@ There are a few common errors that one might run into when trying to execute Ans
 
 To get around this limitation, download and install a later version of `python for z/OS <https://www.rocketsoftware.com/zos-open-source>`_ (2.7.13 or 3.6.1) that represents strings internally as ascii.  Version 2.7.13 is verified to work.
 
+When `pipelining = False` in `/etc/ansible/ansible.cfg` then Ansible modules are transfered in binary mode via sftp however execution of python fails with 
+
+.. error::
+    SyntaxError: Non-UTF-8 code starting with \'\\x83\' in file /a/user1/.ansible/tmp/ansible-tmp-1548232945.35-274513842609025/AnsiballZ_stat.py on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
+  
+To fix it set `pipelining = True` in `/etc/ansible/ansible.cfg`.
+
 .. error::
-  /usr/bin/python: EDC5129I No such file or directory
+    /usr/bin/python: EDC5129I No such file or directory
 
 To fix this set the path to the python installation in your inventory like so::
```

Comment:
```suggestion
  To fix this set the path to the python installation in your inventory like so::
```

Reply:
There is another Error listed below this but isn't showing up in this set of diffs. Can you please make the same adjustments to that one? 
 - add an intro bullet
- indent the error and related lines by 2 spaces
- indent the suggested fix by 2 spaces

Reply:
sure